src/config/babelrc.js

```diff
@@ -55,7 +56,7 @@ module.exports = () => ({
       ['react', 'preact'],
       [
         require.resolve('@babel/preset-react'),
-        {pragma: isPreact ? 'React.h' : undefined},
+        {pragma: isPreact ? ifDep('preact-compat', 'React.h', 'h') : undefined},
```

Comment:
Maybe this should be `React.createElement` if `preact-compat` is installed? 🤔

Reply:
Can do ... but I think the compiled output is probably more readable with `React.h`? I guess React developers are used to seeing `React.createElement` in the output so I don't know. Your call :)

Reply:
Well, we should really care about Preact developers in this case. Which would be more sensible do you think?

Reply:
I'd say the Preact case is covered, `h` is the default expectation. If you're using `preact-compat` you might be using a standard React workflow and using Preact to reduce size, or you're using `preact-compat` to integrate some third-party library written in React. I'd say `React.createElement` for the former and `React.h` for the latter. 

If I assume consumers of `kcd-scripts` are developers building JS libraries then I'd pick `React.h` because it's easier to read, and I think it's generally well known by most library authors using JSX that `h` (from the original idea hyperscript) is the standard practice.